### PR TITLE
Olivia logout

### PR DESCRIPTION
**Pull request by oliviaw12**
* Finished overall implementation of the logout use case which outputs a message informing user they have logged out
* Modified the views so that once logout is complete, user is taken back to the login screen where they are prompted to login again.

### CODE REVIEW

File: src/use_case/matching/logout/LogoutInteractor.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    final LogoutOutputBoundary logoutPresenter;
+
+    public LogoutInteractor(LogoutUserDataAccessInterface userDataAccessInterface,
+                           LogoutOutputBoundary logoutOutputBoundary) {
+        this.userDataAccessObject = userDataAccessInterface;
+        this.logoutPresenter = logoutOutputBoundary;
+    }
+    @Override
+    public void execute() {
+       // String loggedoutUser = userDataAccessObject.get(loginInputData.getUsername());
```
Comment (not-ryan-ning) [inline src/use_case/matching/logout/LogoutInteractor.java]: Is this code redundant? I'm not sure why it is commented out here

Comment (oliviaw12) [reply src/use_case/matching/logout/LogoutInteractor.java]: Yes, I forgot to delete this line of code as it was no longer necessary for my implementation. Thank you!

**Review by ShihHsin0723 — APPROVED** [review]
LogoutOutputData does not return anything and is not necessary. Maybe you can consider deleting it?

File: src/use_case/matching/logout/LogoutInteractor.java (hunk 1 above)
Comment (not-ryan-ning) [reply src/use_case/matching/logout/LogoutInteractor.java]: Ok great!

**Pull request merged**